Mathlib/SetTheory/Game/Impartial.lean

```diff
@@ -133,61 +124,97 @@ theorem mk'_add_self : (⟦G⟧ : Game) + ⟦G⟧ = 0 :=
   game_eq (add_self G)
 
 /-- This lemma doesn't require `H` to be impartial. -/
-theorem equiv_iff_add_equiv_zero (H : PGame) : (H ≈ G) ↔ (H + G ≈ 0) := by
+theorem equiv_iff_add_equiv_zero (H : PGame) : H ≈ G ↔ H + G ≈ 0 := by
   rw [equiv_iff_game_eq, ← add_right_cancel_iff (a := ⟦G⟧), mk'_add_self, ← quot_add,
     equiv_iff_game_eq, quot_zero]
 
 /-- This lemma doesn't require `H` to be impartial. -/
-theorem equiv_iff_add_equiv_zero' (H : PGame) : (G ≈ H) ↔ (G + H ≈ 0) := by
+theorem equiv_iff_add_equiv_zero' (H : PGame) : G ≈ H ↔ G + H ≈ 0 := by
   rw [equiv_iff_game_eq, ← add_left_cancel_iff, mk'_add_self, ← quot_add, equiv_iff_game_eq,
     Eq.comm, quot_zero]
 
-theorem le_zero_iff {G : PGame} [G.Impartial] : G ≤ 0 ↔ 0 ≤ G := by
+variable {G}
+
+@[simp]
+theorem not_equiv_zero_iff : ¬ G ≈ 0 ↔ G ‖ 0 :=
+  ⟨(equiv_or_fuzzy_zero G).resolve_left, Fuzzy.not_equiv⟩
+
+@[simp]
+theorem not_fuzzy_zero_iff : ¬ G ‖ 0 ↔ G ≈ 0 :=
+  ⟨(equiv_or_fuzzy_zero G).resolve_right, Equiv.not_fuzzy⟩
+
+theorem le_zero_iff : G ≤ 0 ↔ 0 ≤ G := by
   rw [← zero_le_neg_iff, le_congr_right (neg_equiv_self G)]
 
-theorem lf_zero_iff {G : PGame} [G.Impartial] : G ⧏ 0 ↔ 0 ⧏ G := by
+theorem lf_zero_iff : G ⧏ 0 ↔ 0 ⧏ G := by
   rw [← zero_lf_neg_iff, lf_congr_right (neg_equiv_self G)]
 
+@[simp]
+theorem le_zero_iff_equiv : G ≤ 0 ↔ G ≈ 0 :=
```

Comment:
This might have an overly strong discrimination key, should I make it `scoped simp`?